sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient.cs

```diff
@@ -0,0 +1,969 @@
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="userId">Target user Id.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response AddUserToGroup(string group, string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.AddConnectionToGroup");
+            scope.Start();
+            try
+            {
+                return RestClient.AddUserToGroup(hub, group, userId, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Remove a user from the target group.</summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="userId">Target user Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<Response> RemoveUserFromGroupAsync(string group, string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.RemoveUserFromAllGroupsAsync");
+            scope.Start();
+            try
+            {
+                return await RestClient.RemoveUserFromGroupAsync(hub, group, userId, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Remove a user from the target group.</summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="userId">Target user Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response RemoveUserFromGroup(string group, string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.RemoveUserFromAllGroups");
+            scope.Start();
+            try
+            {
+                return RestClient.RemoveUserFromGroup(hub, group, userId, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        private static Models.WebPubSubPermission ToPermission(WebPubSubPermission permission)
```

Comment:
Why not make the `Models.WebPubSubPermission` type public?  If it's just because of the namespace, do https://github.com/azure/autorest.csharp#change-a-model-namespace.  If we need something like this, please move it to the same file as the enum so it's harder for them to get out of date.

Reply:
The Models permission is an extensible enum :-)

Reply:
Ah, then we change the swagger or add a README transform like https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/autorest.md#fix-georeplication.  It's way better than having to update these by hand every time the swagger changes.